fabric-biome-api-v1/src/main/java/net/fabricmc/fabric/api/biome/v1/BiomeSelectionContext.java

```diff
@@ -135,4 +134,12 @@ default boolean hasBuiltInStructure(ConfiguredStructureFeature<?, ?> configuredS
 	 * from this biomes feature list.
 	 */
 	Optional<RegistryKey<ConfiguredStructureFeature<?, ?>>> getStructureKey(ConfiguredStructureFeature<?, ?> configuredStructure);
+
+	/**
+	 * Tries to determine whether this biome generates in a specific dimension, based on the dimension options
+	 * present in the dynamic registries.
+	 * <p/>
```

Comment:
```suggestion
	 * <p>
```
Makes more sense logically but doesn't matter in the output

Reply:
Actually the comment itself is wrong because the dimension options are not in the dynamic registries 😁

Reply:
This is kinda the "big change" of this PR. The biome selectors can now also operate on information found in the properties of the world-save being loaded. I'll have to correct the comment.

Reply:
Fixed the comments